docs/kb/config.md

```diff
@@ -8,7 +8,7 @@ title: config.yml reference
 
 The `config.yml` file generated in your blockchain folder uses key-value pairs to describe the development environment for your blockchain.
 
-Only a default set of parameters is provided. If more nuanced configuration is required, you can add these parameters to the `config.yml` file.
+Only a default set of parameters is provided. If more nuanced configuration is required, you can add and update these parameters in the `config.yml` file.
```

Comment:
my new blockchain already had some of these sections, so users will add entries to the existing sections, they won't have to add new